src/main/java/no/nav/veilarbaktivitet/db/dao/AktivitetDAO.java

```diff
@@ -17,7 +17,6 @@
 import static org.slf4j.LoggerFactory.getLogger;
 
 @Repository
```

Comment:
Stusser på at denne er `@Repository`, men ikke direkte relevant for denne PR'en.

Reply:
Hva mener du den burde vert?

Reply:
Egentlig bare en `@Component`, ikke at det har så veldig mye å si. Bare at `@Repository` har [en bestemt use case](https://www.baeldung.com/spring-component-repository-service) i Spring og at de andre DAO'ene er `@Component`.